mava/systems/ippo/networks.py

```diff
@@ -326,7 +331,10 @@ def critic_fn(inputs: jnp.ndarray) -> networks_lib.FeedForwardNetwork:
         Returns:
             FeedForwardNetwork class
         """
-        critic_network = hk.Sequential(
+        critic_layers = []
+        if observation_network:
+            critic_layers.extend([observation_network])
```

Comment:
Same with this line.

Reply:
For consistency, I think it is better to have everything as an extends.

Reply:
See my above comment.